docker/build-and-push-to-dockerhub.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
It doesn't look like this script is being used (the container is being pushed to the github registry in the workflow). Is there a reason to keep this?